integration-tests/features/components.feature

```diff
@@ -21,6 +21,15 @@ Feature: Components API V1
     When I search for component foobar with authorization token
     Then I should get 200 status code
 
+  @requires_authorization_token
+  Scenario: Check the component search API entry point for the component that does not exist
+    Given System is running
+    Given Component search service is running
+    When I acquire the authorization token
+    Then I should get the proper authorization token
+    When I search for component the_strange_component_that_does_not_exist with authorization token
+    Then I should get 200 status code
```

Comment:
Maybe missing a check that the endpoint signalizes no component found (empty results?)?

Reply:
Yeah, it would require a new test definition in component_analysis.py. I'm working on it. TY @fridex